portal-api/src/v1/validation/fields/date.js

```diff
@@ -1,40 +1,6 @@
-const moment = require('moment');
 const { hasValue } = require('../../../utils/string');
 
-exports.dateHasAllValues = (day, month, year) => {
-  const hasDay = hasValue(day);
-  const hasMonth = hasValue(month);
-  const hasYear = hasValue(year);
-
-  const hasValues = (hasDay && hasMonth && hasYear);
-  return hasValues;
-};
-
-exports.dateHasSomeValues = (day, month, year) => {
-  const hasDay = hasValue(day);
-  const hasMonth = hasValue(month);
-  const hasYear = hasValue(year);
-
-  const hasSomeValues = (hasDay || hasMonth || hasYear);
-  return hasSomeValues;
-};
-
-exports.dateIsInTimeframe = (day, month, year, start, end) => {
-  const formattedDate = moment(`${year}-${month}-${day}`);
-  const isInTimeframe = moment(formattedDate).isBetween(start, end, 'days', '[]');
-
-  if (isInTimeframe) {
-    return true;
-  }
-  return false;
-};
-
-exports.dateValidationText = (
-  fieldTitle,
-  dayValue,
-  monthValue,
-  yearValue,
-) => {
+exports.dateValidationText = (fieldTitle, dayValue, monthValue, yearValue) => {
```

Comment:
I'm of two minds whether we want to keep a validation in here or not -- Any strong reason you removed it?

Reply:
I disliked `dateIsInTimeframe` (the implementation in particular).

Removing the `dateHasAllValues` was because I didn't think it added much & gave the illusion of validating the values. Happy to rever this if you disagree